app/lib/features/search/pages/search_page.dart

```diff
@@ -26,7 +26,7 @@ class SearchPage extends ConsumerWidget {
           }
         }
       },
-      expand: true,
+      expand: false,
```

Comment:
wanna explain this change?

Reply:
I think this was causing UI overflow when the text field gets focus. Maybe not, I will see that in fresh PR. 👍🏻